lib/global.neon

```diff
@@ -142,6 +142,14 @@ DECLARE EXCEPTION Utf8Encoding
  |%
 DECLARE EXCEPTION ValueRange
 
+%|
+ |  Exception: StringIndex
+ |
+ |  An attempt was made to read or reference a string beyond with a length 
```

Comment:
This is still too specific. See my previous suggestion. Consistency is important.